apps/sample-app/src/screens/demos/MarkerMapScreen.tsx

```diff
@@ -137,9 +138,9 @@ export const MarkerMapScreen = () => {
   const { showSnackbar } = useSnackbar();
 
   const omhMapRef = useRef<OmhMapViewRef | null>(null);
-  const [supportedFeatures, setSupportedFeatures] = useState(
-    getSupportedFeatures()
-  );
+  const [supportedFeatures, setSupportedFeatures] = useState<ReturnType<
```

Comment:
What was the motivation to this change?

Reply:
I needed to make it nullable, to know if `supportedFeatures` were initialized with the actual correct provider coming from the `onMapLoaded`. This nullability is leverage in line 323. If you test the code before this change, you will notice that the draggable checkbox is always false by default, where it should be true - basically useEffect from line 323 was triggered with default, no provider `supportedFeatures` values, set to false, and then when map was loaded it was not resetting it to true, because of the condition in this effect. We could change the condition, but then it could have a side effect of potentially changing the user choice.